index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Nelson Mandela</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <main id="main">
```

Comment:
You should have used `<header>` as stated on the [Semantic](https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md)

Reply:
Please, always make sure to phrase the comments politely, as suggestions instead of orders. For example:

* This is a polite suggestion: *"You should have used <header> as stated on the [Semantic](https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md)"*
* This reads as an order: *"Kindly use a header tag here for better semantics. Check out [this article](https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md) for more about semantics"*